src/test/java/core/basesyntax/files/FileReaderImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.files;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String fileName = "temp.csv";
+    private static final Path path = Paths.get(fileName);
+    private static final FileReader fileReader = new FileReaderImpl();
```

Comment:
Initialize this field inside of method with annotation @BeforeClass

Reply:
:)